tests/test_attentions.py

```diff
@@ -81,17 +81,21 @@ def test_order_invariance(
     )
 
     # Check that a shuffled input produces the same results
-    inputs = torch.rand(BATCH, SEQ, MODEL, device=device)
-    shuffle = torch.randperm(inputs.shape[1])
-    inputs_shuffled = inputs[:, shuffle, :].clone()
+    seqs = [SEQ, SEQ - 16] if attention_name != "blocksparse" else [SEQ]
```

Comment:
check for all the attentions that we can pass smaller sequences than initially declared